cypress/integration/seededFlows/profileFlows/followOrganisation.spec.js

```diff
@@ -12,24 +12,21 @@ describe('Follow user from profile page', () => {
   it('follows and unfollows an organisation', () => {
     cy.intercept('/follows').as('followsRequest');
 
-    cy.findByRole('button', {
-      name: 'Follow organization: Bachmanity',
-    }).click();
-    cy.wait('@followsRequest');
-    cy.findByRole('button', { name: 'Unfollow organization: Bachmanity' });
+    cy.findByRole('button', { name: 'Follow organization: Bachmanity' }).as(
+      'followButton',
+    );
 
-    // Check that the update persists after reload
-    cy.visitAndWaitForUserSideEffects('/bachmanity');
-
-    cy.findByRole('button', {
-      name: 'Unfollow organization: Bachmanity',
-    }).click();
-    cy.wait('@followsRequest');
+    cy.get('@followButton').click();
+    // Inner text should now be following
+    cy.get('@followButton').should('have.text', 'Following');
+    cy.get('@followButton').should('have.attr', 'aria-pressed', 'true');
 
-    cy.findByRole('button', { name: 'Follow organization: Bachmanity' });
-
-    // Check that the update persists after reload
+    // Check that state is persisted on refresh
     cy.visitAndWaitForUserSideEffects('/bachmanity');
-    cy.findByRole('button', { name: 'Follow organization: Bachmanity' });
```

Comment:
need to add this back in, and check the correct aria-pressed state has been persisted on refresh